plugin.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: Newsletter Builder
  * Plugin URI: https://github.com/alleyinteractive/wp-newsletter-builder
  * Description: Interface to manage email newsletters
- * Version: 0.3.22
+ * Version: 0.3.23
```

Comment:
I forgot to mention earlier that we now have an npm script that will create releases! You can just run `npm run release` and it'll walk you through bumping the version!

Reply:
Awesome! Thanks!